portlet/exoadmin/src/main/java/org/exoplatform/organization/webui/component/UIAccountEditInputSet.java

```diff
@@ -147,15 +147,21 @@ public boolean save(OrganizationService service) throws Exception {
         Query query = new Query();
         String email = getUIStringInput("email").getValue();
         query.setEmail(email);
-        if (!oldEmail.equals(email) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
+        if (((oldEmail != null && !oldEmail.equals(email)) || oldEmail == null) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
```

Comment:
Could be simplified by `if ((oldEmail == null || !oldEmail.equals(email)) && ...`.